packages/storybook/stories/va-combo-box-uswds.stories.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const comboBoxDocs = getWebComponentDocs('va-combo-box');
+
+export default {
+  title: 'Components/Combo Box USWDS',
+  id: 'uswds/va-combo-box',
+  parameters: {
+    componentSubtitle: 'va-combo-box web component',
+    docs: {
+      page: () => <StoryDocs storyDefault={Default} data={comboBoxDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  label: 'Select a fruit',
+  name: 'fruit',
+  value: '',
+  required: false,
+  error: undefined,
+  options: [
+    <option value="apple">Apple</option>,
+    <option value="banana">Banana</option>,
+    <option value="blackberry">Blackberry</option>,
+    <option value="blueberry">Blueberry</option>,
+    <option value="boysenberry">Boysenberry</option>,
+    <option value="cherry">Cherry</option>,
+    <option value="crab apple">Crab Apple</option>,
+    <option value="cranberry">Cranberry</option>,
+    <option value="custard apple">Custard apple</option>,
+    <option value="date">Date</option>,
+    <option value="elderberry">Elderberry</option>,
+    <option value="fig">Fig</option>,
+    <option value="gooseberry">Gooseberry</option>,
+    <option value="mango">Mango</option>,
+    <option value="mangosteen">Mangosteen</option>,
+    <option value="marionberry">Marionberry</option>,
+    <option value="pineapple">Pineapple</option>,
+    <option value="raspberry">Raspberry</option>,
+    <option value="rambutan">Rambutan</option>,
+    <option value="starfruit">Starfruit</option>,
+    <option value="strawberry">Strawberry</option>
+  ],
+};
+
+const Template = ({
+  label,
+  name,
+  value,
+  required,
+  error,
+  hint,
+  options,
+  placeholder,
+  disabled,
+}) => {
+
+  return (
+    <va-combo-box
+      label={label}
+      name={name}
+      value={value}
+      required={required}
+      error={error}
+      hint={hint}
+      placeholder={placeholder}
+      disabled={disabled}
+    >
+      {options}
+    </va-combo-box>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = { ...defaultArgs };
+Default.argTypes = propStructure(comboBoxDocs);
+
+export const WithDefaultValue = Template.bind({});
+WithDefaultValue.args = {
+  ...defaultArgs,
+  value: 'banana',
+};
+
+export const WithError = Template.bind({});
+WithError.args = {
+  ...defaultArgs,
+  error: 'This field contains an error.',
+};
+
+export const Required = Template.bind({});
+Required.args = {
+  ...defaultArgs,
+  required: true,
+};
+
+export const WithPlaceholderText = Template.bind({});
+WithPlaceholderText.args = {
+  ...defaultArgs,
+  placeholder: '--Select--',
+};
+
+export const WithHintText = Template.bind({});
+WithHintText.args = {
+  ...defaultArgs,
+  hint: 'This is example hint text',
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  ...defaultArgs,
+  value: 'elderberry',
+  disabled: true,
```

Comment:
I don't think we want to advertise that there is a disabled state. There are a11y concerns with this functionality. 

Or maybe we don't want to have this functionality in the component at all? I will let the design and a11y reviewers weigh in on that though.

![Screenshot 2024-11-01 at 9 28 50 AM](https://github.com/user-attachments/assets/6bb7b34a-8bb3-48b4-8865-2bc7716b87e1)

Reply:
understood, will remove from the doc but will keep `disabled` functional as it is now